Classes/Helper/DataCiteXml.php

```diff
@@ -33,7 +33,7 @@ public static function convertFromMetsXml($metsXml)
         $metsXml->registerXPathNamespace('slub', 'http://slub-dresden.de/');
 
         // doi
-        $metsDoi = $metsXml->xpath("//mods:identifier[@type='qucosa:doi']");
```

Comment:
What was the reason we used qucosa:doi before? And why did this change?

Reply:
I'm sorry, but I can't remember.